kin-ecosystem-sdk/src/main/java/com/kin/ecosystem/data/order/OrderRepository.java

```diff
@@ -37,20 +38,16 @@
 
 public class OrderRepository implements OrderDataSource {
 
-	private static final String TAG = OrderRepository.class.getSimpleName();
-	private static final String ORIGIN_EXTERNAL = "external";
-
 	private static OrderRepository instance = null;
 	private final OrderDataSource.Local localData;
 	private final OrderDataSource.Remote remoteData;
 
-	private final OfferDataSource offerRepository;
 	private final BlockchainSource blockchainSource;
 	private final EventLogger eventLogger;
 
 	private OrderList cachedOrderList;
 	private ObservableData<OpenOrder> cachedOpenOrder = ObservableData.create();
-	private ObservableData<Order> completedOrder = ObservableData.create();
+	private ObservableData<Order> orderWatcher = ObservableData.create();
```

Comment:
orderObservable could be better naming IMO

Reply:
be my guest

Reply:
not IMO :)

Reply:
ok